README.md

```diff
@@ -1,152 +1,4 @@
-# Orbiter The Meta Cluster Manager
+# Orbos the GitOps Cloud-Automation Project
 
-[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
-[![Release](https://github.com/caos/orbiter/workflows/Release/badge.svg)](https://github.com/caos/orbiter/actions)
-[![license](https://badgen.net/github/license/caos/orbiter/)](https://github.com/caos/orbiter/blob/master/LICENSE)
-[![release](https://badgen.net/github/release/caos/orbiter/stable)](https://github.com/caos/orbiter/releases)
-[![Go Report Card](https://goreportcard.com/badge/github.com/caos/orbiter)](https://goreportcard.com/report/github.com/caos/orbiter)
```

Comment:
Why do we not reuse those?

Reply:
they are still there, just currently moved to docs/orbiter.md

Reply:
Reuse License too please

Reply:
I would delete them, and place them `including` the license on the root. Otherwise I find it confusing

Reply:
done